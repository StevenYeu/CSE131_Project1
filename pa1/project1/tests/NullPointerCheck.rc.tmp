Error, "project1/tests/NullPointerCheck.rc": 
  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
