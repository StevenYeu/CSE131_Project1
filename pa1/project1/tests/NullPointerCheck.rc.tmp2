  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
