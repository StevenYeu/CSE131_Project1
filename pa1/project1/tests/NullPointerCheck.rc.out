Error, "/tmp/cs131_testrunner_DJ0qMF/file.rc": 
  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
